spec/services/induction/amend_participant_cohort_spec.rb

```diff
@@ -126,15 +126,22 @@
         end
       end
 
-      context "when the participant has declarations" do
-        before do
-          allow_any_instance_of(described_class).to receive(:participant_declarations).and_return(true)
-        end
+      %i[submitted eligible payable paid].each do |declaration_state|
```

Comment:
could you add a test for an existing declaration that's clawed back or voided just to show that it goes through?